lib/chai/config.js

```diff
@@ -50,6 +50,21 @@ module.exports = {
    * @api public
    */
 
-  truncateThreshold: 40
+  truncateThreshold: 40,
 
+  /**
+   * ### config.useProxy
+   *
+   * User configurable property, define if chai will use a Proxy to throw
```

Comment:
Agreed on "defines" over "define".

Also maybe be a bit more explicit on the purpose of this feature. For example adding onto the end:

"...non-existent property is read, which protects users from typos when using property-based assertions."